README.md

```diff
@@ -380,6 +377,17 @@ configured with the following attributes:
   shown as a dropdown to the user.  It must match the list of port numbers in
   both order and number.
 
+You can also use the following nested XML tags:
+
+* `provider`: (Optional) A list of OpenStack providers configured in the
+  platform.  `name` must match one of the providers in the XBlock
+  configuration, `capacity` specifies how many simultaneous environments should
+  be launched in that provider before trying the next one, and `environment`
+  specifies a content store path to a Heat environment file specifying template
+  parameters for that provider.
+
+* `test`: (Optional) A list of scripts that will run against the lab
```

Comment:
Same as above. Suggest to clarify that you can define _multiple_ `<test>` elements which then _constitute_ a list of tests, rather than a _single_ `<test>` element that _contains_ a list of tests.